src/Core/WPF/NullToBooleanConverter.cs

```diff
@@ -0,0 +1,43 @@
+﻿/*
+ * SonarLint for Visual Studio
+ * Copyright (C) 2016-2020 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+using System;
+using System.Globalization;
+using System.Windows.Data;
+
+namespace SonarLint.VisualStudio.Core.WPF
+{
+    /// <summary>
+    /// Converter that returns true when given value is null, and false otherwise.
+    /// </summary>
+    [ValueConversion(typeof(object), typeof(bool))]
+    public class NullToBooleanConverter : IValueConverter
```

Comment:
I really tried to bind to `LineNumber.HasValue` but it just doesn't work. It doesn't seem to be supported in WPF and I couldn't find anyone doing it.
I also tried using `TargetNullValue` but that didn't work either.
This converter feels completely redundant but I couldn't make it work purely in the xaml.